.github/workflows/linting.yml

```
@@ -0,0 +1,16 @@
+name: Linting
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          wget https://github.com/rhysd/actionlint/releases/download/v1.7.1/actionlint_1.7.1_linux_amd64.tar.gz
+          tar xvf actionlint_1.7.1_linux_amd64.tar.gz
+          chmod +x actionlint
+          ./actionlint
```

I've added actionlint to the stellar/binaries repo so that we can use a build that we can have some supply chain confidence\* of. 


Any binary from the stellar/binaries repo can be installed with:

```suggestion
      - uses: stellar/binaries@v28
        with:
          name: actionlint
          version: v1.7.1
      - run: actionlint
```

_\* We know the build can't change because the go toolchain ensures it matches the source tagged with that version the first time the version was seen by any go compiler._